install.sh

```diff
@@ -321,177 +368,177 @@ if ($INSTALL_SUB) && !(ls $CATKIN_DIR/src | grep --quiet "Sub8"); then
+	sudo apt-get update -qq
+	sudo apt-get install -qq gcc-arm-none-eabi
+
+	# Visualization
+	sudo apt-get install -qq python-progressbar
+
 	instlog "Installing Navigator ROS dependencies"
 
 	# Serial communications
-	sudo apt-get install -qq ros-indigo-rosserial ros-indigo-rosserial-python ros-indigo-rosserial-arduino
+	sudo apt-get install -qq ros-$ROS_VERSION-rosserial
+	sudo apt-get install -qq ros-$ROS_VERSION-rosserial-arduino
 
 	# Thruster driver
-	sudo apt-get install -qq ros-indigo-roboteq-driver
+	sudo apt-get install -qq ros-$ROS_VERSION-roboteq-driver
 
-	instlog "Installing Navigator dependencies from source"
+	instlog "Cloning Navigator Git repositories that need to be built"
 
-	# Open Dynamics Engine
-	rm -rf /tmp/pyode-build
-	mkdir -p /tmp/pyode-build
-	cd /tmp/pyode-build
-	sudo apt-get build-dep -qq python-pyode
-	sudo apt-get remove -qq python-pyode
-	apt-get source --compile -qq python-pyode
-	sudo dpkg -i python-pyode_*.deb
+	# Software to interface with MIL hardware
+	ros_git_get https://github.com/uf-mil-archive/hardware-common
 
-	# Message types
-	sudo apt-get install -qq ros-indigo-tf2-sensor-msgs ros-indigo-tf2-geometry-msgs
+	# Required steps to build and install lqRRT
+	ros_git_get https://github.com/jnez71/lqRRT.git
+	sudo python $CATKIN_DIR/src/lqRRT/setup.py build
+	sudo python $CATKIN_DIR/src/lqRRT/setup.py install
 
-	# Pulling large project files from Git-LFS
+	# Pull large project files from Git-LFS
 	instlog "Pulling large files for Navigator"
 	cd $CATKIN_DIR/src/Navigator
 	git lfs pull
-
-	instlog "Cloning Navigator Git repositories that need to be built"
-	ros_git_get https://github.com/jnez71/lqRRT.git
-	ros_git_get https://github.com/gareth-cross/rviz_satellite.git
-
-	# Required steps to build and install lqRRT
-	sudo python $CATKIN_DIR/src/lqRRT/setup.py build
-	sudo python $CATKIN_DIR/src/lqRRT/setup.py install
 fi
 
 
-#==============================#
-# Sub8 Dependency Installation #
-#==============================#
-
-if ($INSTALL_SUB); then
-	instlog "Installing Sub8 dependencies from the Ubuntu repositories"
-
-	# Optical character recognition
-	sudo apt-get install -qq tesseract-ocr
-
-	# Hardware drivers
-	sudo apt-get install -qq libusb-1.0-0-dev
-
-	instlog "Installing Sub8 ROS dependencies"
-
-	# 3D Mouse
-	sudo apt-get install -qq ros-indigo-spacenav-node
-
-	instlog "Installing Sub8 dependencies from Python PIP"
-
-	# Libraries needed by the hydrophone board
-	sudo pip install -q -U crc16
-fi
-
 #=========================#
 # Bashrc Alias Management #
 #=========================#
 
-# Copies the MIL aliases file to the machine's home directory file and sources it
-cat $CATKIN_DIR/src/Navigator/scripts/mil_aliases.sh > $ALIASES_FILE
-source $ALIASES_FILE
-if !(cat $BASHRC_FILE | grep --quiet "source $ALIASES_FILE"); then
-	echo "source $ALIASES_FILE" >> $BASHRC_FILE
+SWC_ALIASES=$CATKIN_DIR/src/software-common/scripts/bash_aliases.sh
```

Comment:
Since these files don't exist yet, they should be added as empty files to the other repo's or their sourcing should be commented out for the time being.